gettsim/social_insurance_contributions/ges_krankenv.py

```diff
@@ -258,7 +316,8 @@ def ag_beitr_ges_krankenv_midi_job(
     bruttolohn_m_in_gleitzone = bruttolohn_m.loc[in_gleitzone]
     return (
         bruttolohn_m_in_gleitzone
-        * soz_vers_beitr_params["soz_vers_beitr"]["ges_krankenv"]["ag"]
+        * soz_vers_beitr_params["soz_vers_beitr"]["ges_krankenv"]["allg"]
```

Comment:
Is it correct that the "Zusatzbeitrag" is not considered here under all regimes?

Reply:
This is the employer part, why should it depend on "zusatzbeitrag"?